docs/src/content/docs/de/reference/frontmatter.md

```diff
@@ -1,12 +1,13 @@
 ---
-title: Frontmatter Referenz
+title: 'Referenz: Frontmatter'
```

Comment:
This seems to be a "style" change from the other references. Is this expected? Do you think such change should be applied to other references?

As a non-native speaker, I unfortunately cannot really comment on this change so let me know.

Reply:
OMG!!! I just discovered that it is possible to set a different label in the global sidebar in Starlight.

Hence I think I will remove all the duplicate mentions of "Referenz" in the sidebar, f.e. the sidebar label of the Frontmatter would be "Frontmatter" only because the folder already says "Referenzen". 

But on the actual side, the title could either be consistent with "Referenz: Frontmatter" or "Frontmatter Refernz". What would you like more @HiDeoo ?

Reply:
I think this may be one case where another native speaker may be required. If we simply take a look at the English version, "Reference" is repeated but maybe that does not convert well to some other languages and I definitely cannot be the judge of that.

<img width="274" alt="image" src="https://github.com/user-attachments/assets/e772460b-a5ae-4602-8460-d9caeca0e1b9">

Reply:
Totally fair. I would say, it's also possible in German, grammatically and so, to include "Referenz" in every page title somehow. 

But what makes it kinda ugly is the fact, that in German words can theoretically be chained together indefinitely, e.g. "Konfigurationsreferenz" while in English the words are still separated. This leads me to believe that the English version is much easier to read, which could be countersteered, by not including the reference part in the German version.

That's the reason I would like to make this rather huge style change in the German version. But let's wait for some other native speakers to jump right into this discussion! 😊

Reply:
Discussed the proposed change with Chris and Hippo and they're both on board with the suggested change so we're going to go ahead with it. Thanks a lot for the patience until we figured this out and also suggesting the improvement.